code/modules/antagonists/heretic/heretic_antag.dm

```diff
@@ -35,6 +35,8 @@
 	var/passive_gain_timer = 20 MINUTES
 	/// Assoc list of [typepath] = [knowledge instance]. A list of all knowledge this heretic's reserached.
 	var/list/researched_knowledge = list()
+	/// The organ slot we place our Living Heart in.
+	var/heart_organ_slot = ORGAN_SLOT_HEART
```

Comment:
Maybe rename this to `living_heart_organ_slot` to differentiate it from the normal heart slot